build_helpers/publishPackage.sh

```diff
@@ -1,35 +1,70 @@
 #!/bin/bash
+# example usages:
+# ./publishPackage.sh --branch=v1.0.x
+# ./publishPackage.sh --branch=v1.1.x --latest
+# ./publishPackage.sh --branch=v1.0-beta --beta
+
 set -e
 
 # Defaults that can be overridden in the CLI (except the cookie file)
-export BETA=0
+export BETA=false
```

Comment:
Let's make this a boolean flag and avoid passing a value for the flag.
i.e, instead of `--beta 1` we can simply use `--beta`